app/steps/ui/thankyou/index.js

```diff
@@ -12,9 +12,9 @@ class ThankYou extends Step {
         const ctx = super.getContextData(req);
         ctx.ccdReferenceNumber = '';
         if (req.session.form.ccdCase && req.session.form.ccdCase.id) {
-            ctx.ccdReferenceNumber = req.session.form.ccdCase.id;
+            ctx.ccdReferenceNumber = req.session.form.ccdCase.id.toString();
```

Comment:
Can we add a unit test for this so that we check that we get the expected output if `req.session.form.ccdCase.id` is passed as an integer.

It could be a similar test to `should return the context with the CCD Case ID when present (WITHOUT dashes)` in `test/unit/testThankYouPage.js` but providing an integer instead of a string.